test/e2e/e2e_test.go

```diff
@@ -1017,9 +1017,14 @@ func TestE2E(t *testing.T) {
 	// 1. KUBECONFIG environment variable
 	// 2. /tmp/admin.conf (placed by ci-operator)
 	var confPath string
+	kubeconfigSetButInvalid := false
```

Comment:
i think that our convention was to simply drop `_test.go` during migration. could we change the file to `e2e.go` ?

Reply:
Updated.